config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "appid": "",
   "group": "",
-  "webscene": "19faa71a3bf6468cae35b4fce9393a7d",
+  "webscene": "",
```

Comment:
So should we require a url param and if one isn't there then the error page displays instead of the default scene?